test/system/505-networking-pasta.bats

```diff
@@ -434,43 +434,43 @@ function pasta_test_do() {
 
 ### DNS ########################################################################
 
-@test "External resolver, IPv4" {
-    skip_if_no_ipv4 "IPv4 not routable on the host"
-
-    run_podman '?' run --rm --net=pasta $IMAGE nslookup 127.0.0.1
-
-    assert "$output" =~ "1.0.0.127.in-addr.arpa" \
-           "127.0.0.1 not resolved"
-}
-
-@test "External resolver, IPv6" {
-    skip_if_no_ipv6 "IPv6 not routable on the host"
-
-    run_podman '?' run --rm --net=pasta $IMAGE nslookup ::1
-
-    assert "$output" =~ "1.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.0.ip6.arpa" \
-           "::1 not resolved"
+@test "Basic nameserver lookup" {
+    run_podman run --rm --net=pasta $IMAGE nslookup l.root-servers.net
 }
 
-@test "Local forwarder, IPv4" {
+@test "Default nameserver forwarding" {
     skip_if_no_ipv4 "IPv4 not routable on the host"
 
     # pasta is the default now so no need to set it
     run_podman run --rm $IMAGE grep nameserver /etc/resolv.conf
     assert "${lines[0]}" == "nameserver 169.254.1.1" "default dns forward server"
+}
+
+@test "Custom DNS forward address, IPv4" {
+    skip_if_no_ipv4 "IPv4 not routable on the host"
+
+    local addr=198.51.100.1
 
-    run_podman run --rm --net=pasta:--dns-forward,198.51.100.1 \
-        $IMAGE nslookup 127.0.0.1 || :
-    assert "$output" =~ "1.0.0.127.in-addr.arpa" "No answer from resolver"
+    run_podman run --rm --net=pasta:--dns-forward,$addr \
+        $IMAGE grep nameserver /etc/resolv.conf
+    assert "${lines[0]}" == "nameserver $addr" "custom dns forward server"
+
+    run_podman run --rm --net=pasta:--dns-forward,$addr \
+        $IMAGE nslookup l.root-servers.net $addr
 }
 
-@test "Local forwarder, IPv6" {
+@test "Custom DNS forward address, IPv6" {
     skip_if_no_ipv6 "IPv6 not routable on the host"
 
     # TODO: Two issues here:
     skip "Currently unsupported"
-    # run_podman run --dns 2001:db8::1 \
-    #   --net=pasta:--dns-forward,2001:db8::1 $IMAGE nslookup ::1
+    # local addr=2001:db8::1
+    #
+    # run_podman run --rm --net=pasta:--dns-forward,$addr \
+    #     $IMAGE grep nameserver /etc/resolv.conf
+    # assert "${lines[0]}" == "nameserver $addr" "custom dns forward server"
+    # run_podman run --rm --net=pasta:--dns-forward,$addr \
+    #     $IMAGE nslookup l.root-servers.net $addr
     #
```

Comment:
Since this is `skipped`, WDYT of un-commenting the code, and adding `FIXME` comments describing when this will be supported (or giving hints as to what to look for in future versions of pasta, or Changelog)?

Reply:
Or at the very least saying 202409 or some other indication of what "currently" means?